src/main/resources/content/k8s/all-capabilities-should-be-dropped.json

```diff
@@ -1 +1,8 @@
-{"severity":"high","rule":"$.spec.template.spec.containers[*].securityContext.capabilities.drop exists and !contains all","id":"4682a6f1-2a1b-4f5a-938c-cdd3fa421a63","enabled":true,"resourceType":"k8s","policy":"all capabilities should be dropped"}
\ No newline at end of file
+{
+  "severity": "high",
+  "rule": "$.spec.template.spec.containers[*].securityContext.capabilities.drop exists and !contains all",
+  "id": "4682a6f1-2a1b-4f5a-938c-cdd3fa421a63",
+  "enabled": true,
+  "resourceType": "k8s",
+  "policy": "all capabilities should be dropped"
```

Comment:
description field missing here as well.
Also, the policy name is not sentence case.